pkgs/by-name/az/azure-cli/package.nix

```diff
@@ -413,9 +376,22 @@ py.pkgs.toPythonApplication (
             touch $out
           '';
       };
+
+      generate-extensions = writeScriptBin "${pname}-update-extensions" ''
+        export FILE=extensions-generated.nix
+        echo "# This file is automatically generated. DO NOT EDIT! Read README.md" > $FILE
+        echo "{ mkAzExtension }:" >> $FILE
+        echo "{" >> $FILE
+        ${./query-extension-index.sh} --requirements=false --download --nix --cli-version=${version} \
+          | xargs -n1 -d '\n' echo " " >> $FILE
+        echo "" >> $FILE
+        echo "}" >> $FILE
+        echo "Extension was saved to \"extensions-generated.nix\" file."
+        echo "Move it to \"{nixpkgs}/pkgs/by-name/az/azure-cli/extensions-generated.nix\"."
```

Comment:
Saving it directly to `pkgs/by-name/az/azure-cli/extensions-generated.nix` was problematic, depending on which directory `nix run` is executed it would fail or not. Local file seemed simple enough. (And I got tired of tweaking things.)

Reply:
I can take a look on improving this further after the PR is merged.